backend/typescript/services/implementations/schedulingService.ts

```diff
@@ -525,7 +525,6 @@ class SchedulingService implements ISchedulingService {
         const recurringDonationEndDate = dayjs(
           scheduling.recurringDonationEndDate!,
         );
-        recurringDonationEndDate.startOf("day");
```

Comment:
This line was actually useless since this returns a dayjs object `recurringDonationEndDate` was never being set to start of day. It should have been `recurringDonationEndDate = recurringDonationEndDate.startOf("day")`. But it doesnt matter cuz we dont need it anymore.